src/llmcompressor/args/training_arguments.py

```diff
@@ -26,8 +26,8 @@ class TrainingArguments(HFTrainingArgs):
     output_dir: str = field(
         default="./output",
         metadata={
-            "help": "The output directory where the model predictions and "
-            "checkpoints will be written."
+            "help": "The output directory where the model safetensors, "
+            "recipe, config, and optionally checkpoints will be written."
```

Comment:
This reads as if safetensors, recipe, and config non-optional but the checkpoint is optional

Reply:
Ok I can rephrase